src/vm/gdbjit.cpp

```diff
@@ -947,6 +1094,19 @@ int NotifyGdb::Leb128Encode(int32_t num, char* buf, int size)
     return i;
 }
 
+#ifdef _DEBUG
```

Comment:
It looks like if you use FEATURE_GDBJIT with a DEBUG build a file will be written for each method jit'ed that is in the env var. It looks like this is only for your internal debugging use. Unless you really need it all the time, it should be ifdef'ed with something else.